iocage/lib/ioc_common.py

```diff
@@ -410,11 +408,8 @@ def open_atomic(filepath, *args, **kwargs):
         os.chmod(filepath, 0o644)
 
 
-def get_nested_key(_dict, keys=None):
+def get_nested_key(_dict, keys=[]):
```

Comment:
No. Sorry I missed that one.

The `_callback` is `None` by default so we use ours. I check if it's callable, and if it is I use theirs. If it's not, I use mine. Minus the concatenation of `if silent and level != "EXCEPTION"`, I'd revert the rest of that function.